nutkit/protocol/feature.py

```diff
@@ -13,6 +13,9 @@ class Feature(Enum):
     # is picked up instead or we need to wait until the full pool depletes.
     API_CONNECTION_ACQUISITION_TIMEOUT = \
         "Feature:API:ConnectionAcquisitionTimeout"
+    # The driver offers a method for run a query in a retriable context at the
+    # driver object level
+    API_DRIVER_EXECUTE_QUERY = "Feature:API:Driver:ExecuteQuery"
```

Comment:
it's not the most important change but, `Feature:API:Driver.ExecuteQuery`
GetServerInfo is the exception that proves the rule I guess

Reply:
Agreed. Shame on me for introducing `Feature:API:Driver.ExecuteQuery`. All the others (e.g., `Feature:API:Result.List`, `Feature:API:Driver.VerifyConnectivity`) use dots. Admittedly, there are quite a few that use dots, that shouldn't. Some clean-up may be in order some time later...